iguana/xml_reader.hpp

```diff
@@ -66,6 +92,21 @@ inline void do_read(rapidxml::xml_node<char> *node, T &&t) {
   for (auto &[key, value] : frozen_map) {
     std::string_view str = key.data();
     auto n = node->first_node(key.data());
+
+    if (key == std::string_view("_attribute")) {
```

Comment:
maybe _attribute has conflict with user's member variable, use "___attr" will be better, and you only need to check "___", more efficient.
further more, we can add a macro to disable parse attribute when no attribute to improve performance.